make_deb

```diff
@@ -16,67 +16,51 @@
 ## limitations under the License.
 ##--------------------------------------------------------------------
 ##
-## Author: Ivan Zoratti, Praveen Garg
+## Author: Ivan Zoratti, Praveen Garg, Vaibhav Singhal
 ##
 
 set -e
 
 GIT_ROOT=`pwd`    # The script must be executed from the root git directory
-architecture="none"
-usage="$(basename "$0") {arm} [clean|cleanall]
```

Comment:
restore usage, just remove arm

help option needs it. may be help should also have `-h` as `-h | help`

Reply:
done